src/Symfony/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "async-aws/symfony-bundle",
```

Comment:
what's about `async-aws/aws-bundle` or `async-aws/async-aws-bundle`
For consistency with most of bundles https://packagist.org/search/?q=bundle

`doctrine-bundle`, `twig-bundle`, `flysystem-bundle`, `redis-bundle`, `cords-bundle`, ...

Reply:
Lets do async-aws/async-aws-bundle